recipes/r-rserve/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
@croth1, do you have the time to apply the proposed fix? I'd like to finish level 1 of the R-migration this week.